recipes/libgeoip/meta.yaml

```diff
@@ -10,7 +10,8 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://github.com/maxmind/geoip-api-c/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
-  patch: Makefile.vc.patch
+  patchs:
```

Comment:
This is why I wrote a PR which extended the linter to check for these things :frowning_face: